subsys/emul/emul.c

```diff
@@ -55,6 +55,8 @@ int emul_init_for_bus(const struct device *dev)
 		case EMUL_BUS_TYPE_SPI:
 			emul->bus.spi->target = emul;
 			break;
+		case EMUL_BUS_TYPE_NONE:
```

Comment:
If this is needed here, don't you also need to handle the switch statement at line 68?  Won't you get a warning error in the default case of the 2nd switch?

Reply:
Yes, but the function is only called for child nodes of a bus, so this should never run. I needed the extra case to make the compiler happy.